### PR TITLE
Updated Stack module in database with newer content

### PR DESCRIPTION
**Pull request by rohankaushal123**

### CODE REVIEW

File: Data Structures/Stack.md [hunk 1]
```diff
@@ -1,31 +1,53 @@
...
+2. Linked-list-like (Dynamic and Dis-contiguous)
+In this implementation, a stack is represented using a linked list. Each element of the stack is a node that points to the previous element. This dynamic structure allows for a variable number of elements, making it more memory-efficient but less cache-friendly than the array-like implementation.
+
+## Stack's Common Operations
+Stacks support three primary operations:
+
+1. Push
+The push operation adds an item to the top of the stack. It increments the stack size and stores the item at the newly created top position.
 ```
+Example:
```
Comment (shhossain) [inline Data Structures/Stack.md:19]: keep `Example` above the code blocks.

```
Example:
let stack = [4, 1, 0, 2];
stack.push(5);	stack.push(5);
// stack is now [5, 4, 1, 0, 2]
```
to
Example:
```
let stack = [4, 1, 0, 2];
stack.push(5);	stack.push(5);
// stack is now [5, 4, 1, 0, 2]
```

**Review by shhossain — CHANGES_REQUESTED** [review]

**Comment by rohankaushal123** [comment]
Required change done !
Please review

**Comment by shhossain** [comment]
> Required change done ! Please review

I mean to say keep all examples that are in codeblocks outside. you only changed one.

**Comment by rohankaushal123** [comment]
I've changed all things please recheck

**Comment by rohankaushal123** [comment]
Please recheck there are 3 code blocks , The example is set above all !

**Review by shhossain — APPROVED** [review]
Alright.

**Pull request merged**